spec/support/rb_bug.rb

```diff
@@ -1,5 +1,6 @@
 require 'ffi'
 
+# @api private
```

Comment:
Ohh, I see your flag is not on `lib/mutant.rb` my above comment assumed this location. Still my intention is to set the helpers api to private here.